src/context.rs

```diff
@@ -335,6 +335,23 @@ impl<'a> Context<'a> {
             Duration::from_millis(self.root_config.command_timeout),
         )
     }
+
+    /// Attempt to execute several commands with exec_cmd, return the results of the first that works
+    pub fn exec_cmds_return_first (&self, commands: &[&[&str]]) -> Option<CommandOutput> {
+        let mut attempts_iterator = commands.iter();
```

Comment:
Please try [`find_map`](https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.find_map)

Reply:
Ah-ha, I knew it had to be _somewhere_ in the standard libraries.